libfreerdp/codec/yuv.c

```diff
@@ -171,12 +171,12 @@ BOOL yuv_context_reset(YUV_CONTEXT* WINPR_RESTRICT context, UINT32 width, UINT32
 
 	if (context->useThreads)
 	{
-		const UINT32 pw = (width + TILE_SIZE - width % TILE_SIZE) / TILE_SIZE;
-		const UINT32 ph = (height + TILE_SIZE - height % TILE_SIZE) / TILE_SIZE;
+		const size_t pw = (width + TILE_SIZE - width % TILE_SIZE) / TILE_SIZE;
+		const size_t ph = height + TILE_SIZE - height % TILE_SIZE / TILE_SIZE;
```

Comment:
Why were the braces `()` around `height + TILE_SIZE - height % TILE_SIZE` for `ph` removed, but not for `pw`? Was this accidental? It looks to me, that either they should not be removed at all, or removed for both `ph` and `pw`.

Reply:
@pnowack thx, was really an accident.

Reply:
https://github.com/FreeRDP/FreeRDP/pull/11066